app/views/hyrax/my/collections/index.html.erb

```diff
@@ -20,7 +20,7 @@
 
     <% if can?(:create, Collection) && @collection_type_list_presenter.any? %>
       <% if @collection_type_list_presenter.many? %>
-        <% # modal to select type %>
+        <% # modal tadd-collection-to-collection-deny-modal select type %>
```

Comment:
Not sure I see why the change to this comment.  Looks like it is still the collection type selection modal to me.